makefile-install.include

```diff
@@ -0,0 +1,48 @@
+#
+# This Makefile can be included by images that need to run an install phase,
```

Comment:
I think this file was copied from somewhere (vrnetlab?). We need to attribute that!

Reply:
Done in https://github.com/inmanta/containerlab-vms/pull/1/commits/96d03f6a26eab30f63126f2a2cba0b8be4afb239

Reply:
Is it different in this repo then the original repo? If so I would link to the original repo

Reply:
Done in https://github.com/inmanta/containerlab-vms/pull/1/commits/fcf042eb4e5e65119124ac418af5529ac9c0591e